paper/paper.Rmd

```diff
@@ -217,11 +219,13 @@ There are two main ways to customize it:
 
   - Create new linters (by leveraging functions like `lintr::make_linter_from_xpath()`) tailored to match project- or organization-specific coding standards.
 
+Indeed, `{goodpractice}` [@Padgham2024] bundles a set of custom linters that are not part of the default set of `{lintr}` linters, while `{flint}` [@Bacher2024] provides a way to fix the lints detected by `{lintr}`.
```

Comment:
I am not sure we should mention flint, it is only ~3 months old, not currently on CRAN and it also copies a bunch of lightly modified code from lintr without any attribution in the files or DESCRIPTION. OTOH it is a relatively minor mention, so maybe it is ok.

Reply:
Yeah, I also feel that it should be okay to include this, especially since it was mentioned by the reviewer as well.

Rust-based tools seem to be the future (ruff, polars, etc.), and so I feel like `{flint}` is here to stay and grow.